stl/inc/flat_set

```diff
@@ -403,18 +426,18 @@ public:
 
 private:
     void _Assert_after_sorted_input() const {
-        _STL_ASSERT(_STD is_sorted(begin(), end(), _Get_comp()), "Input was not sorted!");
+        _STL_ASSERT(_STD is_sorted(cbegin(), cend(), _Get_comp()), "Input was not sorted!");
         if constexpr (!_Multi) {
-            _STL_ASSERT(_Is_unique(), "Input was not unique!");
+            _STL_ASSERT(_Is_unique(), "Input was sorted but not unique!");

```

Comment:
As a side note, if just for debug efficiency, `is_sorted && _Is_unique` can be replaced by checking `[0] lt [1] lt [2] ...` in a single pass.